spec.md

```diff
@@ -1711,6 +1761,7 @@ message Error {
     NodeUnpublishVolumeError node_unpublish_volume_error = 8;
     ProbeNodeError probe_node_error = 9;
     GetNodeIDError get_node_id_error = 10;
+    ProbeControllerError probe_controller_error = 11;
```

Comment:
Hi @Kaixiang,

I wonder if this should be reordered so that it's grouped with the other Controller RPCs. Yes, it will be a breaking change due to the field order, but there's no tagged version yet, so I think that is fine.